lib/mvos/service/firebase_service.dart

```diff
@@ -16,10 +21,29 @@ class FirebaseService implements Disposable {
     FirebaseAuth.instance.signInAnonymously();
   }
 
-  Future<void> loginWithEmailAndPassword() async {
+  Future<void> loginWithEmailAndPassword(String email, String password) async {
     logger.info("Login email and password called");
-    FirebaseAuth.instance.signInWithEmailAndPassword(
-        email: "goranesine@gmail.com", password: "Testpassword01");
+    try {
+      FirebaseAuth.instance
+          .signInWithEmailAndPassword(email: email, password: password);
```

Comment:
jel ovdje mora ici neka druga methoda od firebase za login. ova je za sign in.

isto ti fali await

Reply:
Šta ako future baci throw error funkcija ide dalje ili stane?